lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -7,25 +7,21 @@ define((require, exports, module) ->
         placement: "bottom"
         trigger: "manual"
         title: "Formula"
-        container: "#editor"
+        container: editor.container
       }
 
-      show: (jqPopoverContainer, content) ->
-        cursorPosition = $("textarea.ace_text-input").position()
-        jqPopoverContainer.css({
-          top: "#{cursorPosition.top + 24}px"
-          left: "#{cursorPosition.left}px"
-        })
+      show: (jqPopoverContainer, content, position) ->
+        jqPopoverContainer.css(position)
         popoverHandler.options.content = content
         jqPopoverContainer.popover(popoverHandler.options)
         jqPopoverContainer.popover("show")
         return
 
-      hide: (jqPopoverContainer) ->
+      destroy: (jqPopoverContainer) ->
         jqPopoverContainer.popover("destroy")
 
-      isVisible: (jqPopoverElement) ->
-        jqPopoverElement.children(".popover").is(":visible")
+      isVisible: (jqPopoverContainer) ->
+        jqPopoverContainer.data().popover.tip().hasClass("in")
```

Comment:
damn :) maybe it is yet another argument in favor of more simple code :) ? Long line which does something different from whatever you think it does vs short line which does exactly the same what is says, and both are equivalent